### PR TITLE
Kevin fixes

### PR DESCRIPTION
**Pull request by kevinfwl**

### CODE REVIEW

**Review by CalderLund — CHANGES_REQUESTED** [review]
Text to right of button should bold on hover. Button should bold too if possible.

File: src/components/Modals/RequirementOptionsModal.vue [hunk 1]
```diff
@@ -418,19 +421,35 @@
...
+    height: 23px !important;
+}
+
+.modal-course-list-display {
+    margin-top: 1px; /* the way it is displayed does not align event if the height is aligned */
+    display: inline-block;
+    width: 90% !important;
+    vertical-align: bottom !important;
+}
+
```
Comment (haowei99) [inline src/components/Modals/RequirementOptionsModal.vue:466]: this looks pretty hard coded. You sure it works on all screens?

**Review by haowei99 — CHANGES_REQUESTED** [review]
Glitch when selecting around courses in modal. Pmed with video

**Review by CalderLund — DISMISSED** [review]
Add Course popup modal doesn't have the quick add plus functionality.

**Review by haowei99 — APPROVED** [review]
tested lgtm

**Pull request merged**